setup.cfg

```diff
@@ -28,7 +28,7 @@ packages = find:
 package_dir =
     = src
 include_package_data = True
-python_requires = >=3.7
+python_requires = >=3.8
```

Comment:
So, we don't compile wheels for python 3.7 anymore?

Reply:
3.7 is not supported by numpy anymore, so we can drop 3.7, too. importlib.metadata is available in 3.8+, which I use to query the version.

Reply:
https://endoflife.date/python
EOL of 3.7 is officially in 5 months.
Numpy is more strict and dropped support for 3.7 already a while ago. I cannot find the schedule for numpy releases right now.

Reply:
Good, thanks for the information